src/status_im/utils/eip/eip681.cljs

```diff
@@ -0,0 +1,64 @@
+(ns status-im.utils.eip.eip681
+  "Utility function related to [EIP681](https://github.com/ethereum/EIPs/issues/681)
+   e.g. ethereum:0x1234@1/transfer?to=0x5678&value=1e18&gas=5000"
+  (:require [clojure.string :as string]
+            [status-im.constants :as constants]
+            [status-im.utils.money :as money]))
+
```

Comment:
A few lines of comments saying what EIP681 solves and interface it has would be useful. If I click link I end up in a PR with a one line summary and a 10 page discussion.

Reply:
:+1:

Reply:
@jeluard Any chance of this or is it too late now that it is in testing? I figure if it this is just comments it shouldn't impact the actual code :P

Reply:
I added some, do you expect more? Also linked issue should be much better once it's merged.